srtcore/core.h

```diff
@@ -1053,6 +1053,12 @@ class CUDT
     std::pair<bool, time_point> packData(CPacket& packet);
 
     int processData(CUnit* unit);
+    int handleSocketPacketReception(const std::vector<CUnit*>& incoming, bool& w_new_inserted, bool& w_was_sent_in_order, CUDT::loss_seqs_t& w_srt_loss_seqs);
+
+    // Group passed here by void* because in the current imp it's
+    // unused and shall not be used in case when bonding is off
+    time_point getPacketPTS(void* grp, const CPacket& packet);
```

Comment:
Please add Doxygen comments to those functions to understand what are they intended to do.

Reply:
"PTS" is a term not ever mentioned in SRT, although might be familiar to those working with MPEG-2 TS.
However, better to use consistent terminology, e.g. `getPktTsbPdTime`.

Reply:
I think "play time" is also a term used in SRT and as a name is much more readable than PktTsbPd

Reply:
Still, I changed to that name, if that makes things better.